app/src/main/java/org/mozilla/webmaker/javascript/WebAppInterface.java

```diff
@@ -300,6 +300,6 @@ public boolean isDebugBuild() {
      */
     @JavascriptInterface
     public String getSystemLocale() {
-        return Locale.getDefault().toString();
+        return Locale.getDefault().toString().replaceAll("_", "-");
```

Comment:
I also think this should now be change to `getSystemLanguage` since locale standard is underscore and language code is hyphen.